### PR TITLE
Add HPA autoscaling behavior field

### PR DESCRIPTION
**Pull request by taylorshaulis**
# Why do we need this PR?
This PR adds the behavior field to the worker HPA template.  Once added the behavior field will allow users to adjust scale down and scale up windows beyond the default kubernetes settings.

https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#default-behavior

# Changes proposed in this pull request

* Add spec.behavior field to worker HPA to allow for custom scale behavior

# Contributor Checklist

- [ ] Variables are documented in the `README.md`
- [ ] Which branch are you merging into?
    - `master` is for changes related to the current release of the `concourse/concourse:latest` image and should be good to publish immediately
    - `dev` is for changes related to the next release of Concourse (aka unpublished code on `master` in [concourse/concourse](https://github.com/concourse/concourse))

# Reviewer Checklist
> This section is intended for the core maintainers only, to track review progress. Please do not
> fill out this section.
- [ ] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed
- [ ] Is the correct branch targeted? (`master` or `dev`)

### CODE REVIEW

**Comment by taylorshaulis** [comment]
The sub settings for worker.autoscaling are not documented in the README.md.  If that is necessary I can add documentation for the behavior field.

This is something new so hopefully we can just merge into the master branch without needing to back-port or wait for the next concourse release.

File: values.yaml [hunk 1]
```diff
@@ -1894,6 +1894,16 @@ concourse:
       # for this to work you need to configure the metric eg. with prometheus adapter
 #      maxReplicas: 1
 #      minReplicas: 1
+#     behavior:
+#        scaleDown:
```
Comment (xtremerui) [inline values.yaml:1898]: could you please fix the indent here?

**Review by xtremerui — CHANGES_REQUESTED** [review]
Thx for the PR. Just small fix then its good.

File: values.yaml (hunk 1 above)
Comment (taylorshaulis) [reply values.yaml:1898]: Nice catch :)

How does it look now?

**Review by xtremerui — APPROVED** [review]

**Pull request merged**